src/book.rs

```diff
@@ -184,7 +183,7 @@ impl Book {
     async fn r#match(
         &mut self,
         mut order: Order,
-        executioner_address: String,
+        _executioner_address: String,
```

Comment:
can this just be removed?

Reply:
Yes, see my above response to @kumar-ish